spec/services/symphony_client_spec.rb

```diff
@@ -124,4 +124,24 @@
       expect(renew_response).to include error: [error_response]
     end
   end
+
+  describe '#get_bib_info' do
+    before do
+      stub_request(:get, "#{Settings.symws.url}/catalog/bib/key/12345?includeFields=*,callList%7B*,itemList%7B*%7D%7D")
```

Comment:
@cdmo just curious why you did not prefer to use `.with(query: hash_including(includeFields: '*,callList{*,itemList{*}}')`

Reply:
🤷‍♂ no reason, not a thing I thought about. I'll change it